content/about/verslagen/17-18/2018-04-19.md

```diff
@@ -0,0 +1,174 @@
+% Bestuursvergadering 3
+% Isaura Claeys
+% 20/04/2018 - 16:00
+
+# Bestuursvergadering 5
+
+_Datum: 19/04/2018 - 16:00_
+_Status: Gesloten_
+
+**Timetable:**
+
+| Start | Duur | Onderwerp    |
+| ----- | ---- | ------------ |
+| 16:00 | '10  | Intro        |
+| 16:10 | '5   | Financiën    |
+| 16:15 | '5   | Sysadmin     |
+| 16:20 | '5   | Projecten    |
+| 16:25 | '20  | Events       |
+| 16:45 | '15  | Pauze        |
+| 17:00 | '20  | Ereleden     |
+| 17:20 | '20  | Verkiezingen |
+| 17:40 | '20  | Partnerships |
+| 18:00 | ...  | Trivia, ...  |
+
+
+## 0. Verslagen workflow
+Gebruik van hackmd.io: collaboratief verslagen schrijven in markdown. :white_check_mark: 
+ 
+- Goedkeuringen door middel van 'reviews' op PR's
```

Comment:
Na dit hier in actie te zien ben ik volledig gewonnen voor dit idee! Naast de voordelen die hier vermeld worden vind ik het ook erg nuttig dat dit ook een plaats kan zijn voor 'vervolgdiscussies' en -opmerkingen die geen plaats hebben op een bestuursvergadering zelf wegens de beperkte tijd. 

Maar omdat goed nog altijd beter kan:
- Hoe scheiden we productief discours van het verbeteren van spelfouten en andere trivialiteiten?
- Hoe trekken we deze openheid door naar de bestuurswerking at large (ofwel: hoe vermijden we het zeusdrive-fenomeen)?
- Verslagen zijn belangrijk, en het evalueren van voorbije events ook, maar ik betwijfel dat verslagen actief doorzocht gaan worden om die evaluaties van de voorbije jaren te vinden. Waar zijn onze draaiboeken, en hoe zorgen we dat deze feedback (bvb het stukje over de hashcode in dit verslag) op de juiste plek terecht komt, en daar ook gevonden wordt door de volgende verantwoordelijke?

Reply:
**Hoe scheiden we productief discours van het verbeteren van spelfouten en andere trivialiteiten?**
Misschien wanneer een discussie te belangrijk/uitgebreid wordt verhuizen naar een issue?

**Hoe trekken we deze openheid door naar de bestuurswerking at large (ofwel: hoe vermijden we het zeusdrive-fenomeen)?**
Zo veel mogelijk tekstmateriaal in die repository plaatsen? Nu moeten we wel oppassen met gevoelige info zoals de geldstatus, ledenlijst (met emails), ...

In diezelfde 'verslagen' repository kunnen we de EventKnowledgeTransfer™ documenten plaatsen, waar we dan ook feeback van vorige jaren in kunnen verzamelen